pandas/io/html.py

```diff
@@ -846,7 +846,8 @@ def _parser_dispatch(flavor):
 
 
 def _print_as_set(s) -> str:
-    return "{" + "{arg}".format(arg=", ".join(pprint_thing(el) for el in s)) + "}"
+    arg = ", ".join(pprint_thing(el) for el in s)
+    return f"{{{arg}}}"
```

Comment:
is this adding a layer of braces?

Reply:
you might be able to get rid of the pprint_thing, not sure

Reply:
The double brace is how to add a literal brace in an f-string, so looks OK